napari/_app_model/_tests/test_context.py

```diff
@@ -0,0 +1,31 @@
+from unittest.mock import Mock
```

Comment:
I know we have lots of other tests that implicitly call `get_context` and therefore create a `ContextMapping`, but do we think it's worth putting an explicit test here that checks that the app-model `get_context` can be used with `ContextMapping` "invisibly"? That way if something starts failing we won't go chasing red herrings? Idk maybe it's overkill

Reply:
+1 for test

Reply:
test added